src/Toolkit/Escape.php

```diff
@@ -64,7 +71,17 @@ public static function attr($string)
      */
     public static function css($string)
     {
-        return (new Escaper('utf-8'))->escapeCss($string);
+        return static::escaper()->escapeCss($string);
+    }
+
+    /**
+     * Get the escaper instance (and create if needed)
+     *
+     * @return \Laminas\Escaper\Escaper
+     */
+    protected static function escaper()
```

Comment:
I don't know if it makes sense, but for users who want to use different features of the `Escape(r)` class, wouldn't it be better if this was `public`? Of course, unless there will be any security vulnerabilities.

Reply:
I think only the `escapeHtmlAttr()` method is missing. I think we can add that too.

Reply:
We already have the `attr()` method that calls `escapeHtmlAttr()`.

I think the use-case for an exposed escaper instance is limited, so making it `protected` will make the reference and method lists in IDEs less cluttered.